padlockcloud/server.go

```diff
@@ -510,16 +556,18 @@ func (server *Server) Start() error {
 }
 
 // Instantiates and initializes a new Server and returns a reference to it
-func NewServer(log *Log, storage Storage, sender Sender, config *ServerConfig) *Server {
+func NewServer(log *Log, storage Storage, sender Sender, config *ServerConfig, whitelistConfig *WhitelistConfig) *Server {
 	server := &Server{
 		Server: &graceful.Server{
 			Server:  &http.Server{},
 			Timeout: time.Second * 10,
 		},
-		Log:     log,
-		Storage: storage,
-		Sender:  sender,
-		Config:  config,
+		Log:             log,
+		Storage:         storage,
+		Sender:          sender,
+		Config:          config,
+		WhitelistConfig: whitelistConfig,
```

Comment:
`WhitelistConfig` should be a member of `Whitelist`. I.e.
```go
struct Whitelist {
  *WhitelistConfig
  ...
}
```

Reply:
This is a good call, I will make the change.